### PR TITLE
Added sku image to the pricing modal and added image check to the

### PR DESCRIPTION
**Pull request by christopher-kundrat**
thumbnail directive

---
This change is [Reviewable](https://reviewable.io/reviews/ten24/slatwall/5812)

### CODE REVIEW

File: admin/client/src/sku/components/swskuthumbnail.ts [hunk 1]
```diff
@@ -9,13 +9,28 @@ class SWSkuThumbnailController{
...
         if(!angular.isDefined(this.skuData)){
             throw("You must provide a sku to the SWSkuThumbnailController");
         }
-        if(angular.isDefined(this.skuData.imagePath)){
-            this.image = this.skuData.imagePath;
-        }
+
+        fileService.imageExists(this.skuData.imagePath).then(
+            ()=>{
+                this.skuData.imagePath = this.skuData.imagePath;
```
Comment (jasperboyd) [inline admin/client/src/sku/components/swskuthumbnail.ts]: @christopher-kundrat I'm not sure why this would be necessary

Comment (christopher-kundrat) [reply admin/client/src/sku/components/swskuthumbnail.ts]: @jasperboyd - so at this point we haven't verified if the image from swaddskupricemodallauncher.ts actually exists so it's just running a check and setting it to the placeholder if not found.

Comment (jasperboyd) [reply admin/client/src/sku/components/swskuthumbnail.ts]: @christopher-kundrat I'm specifically just talking about this line `this.skuData.imagePath = this.skuData.imagePath;`

Comment (rmarchandTen24) [reply admin/client/src/sku/components/swskuthumbnail.ts]: agreed it would not do anything

Comment (christopher-kundrat) [reply admin/client/src/sku/components/swskuthumbnail.ts]: yeah that's left over from when the image path was being passed in through a different variable. I'll remove it when I resolve the conflict

**Pull request merged**